### PR TITLE
migrated to IntroJs in abacus tutorial

### PR DESCRIPTION
**Pull request by Ujjawal3**
I have migrated abacus activity tutorial from bootstrap-tour to IntroJs
Issue: #1090
[Screencast from 26-10-22 02:48:13 PM IST.webm](https://user-images.githubusercontent.com/99073049/197989668-2c29ca81-88d4-494b-9c5e-c600403a6bbe.webm)

### CODE REVIEW

File: activities/Abacus.activity/lib/tutorial.js [hunk 1]
```diff
@@ -4,82 +4,92 @@ define(["webL10n"], function (l10n) {
...
+	// 					<div class='icon-tutorial-text'>"+l10n.get("TutoEnd")+"</div>\
+	// 				</div>\
+	// 			</div>\
+	// 		</div>",
+	// 		storage: false,
+	// 		backdrop: true,
+	// 		steps: steps
+	// 	});
+	// 	tour.init();
+	// 	tour.start(true);
```
Comment (llaske) [inline activities/Abacus.activity/lib/tutorial.js:78]: Remove commented lines

File: package-lock.json [hunk 2]
```diff
@@ -1,3798 +1,8 @@
...
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
-    "": {
-      "name": "Sugarizer",
-      "version": "1.6.0",
-      "license": "Apache-2.0",
-      "dependencies": {
-        "electron": "~11.5.0",
```
Comment (llaske) [inline package-lock.json:12]: Do not update this file.

**Review by llaske — COMMENTED** [review]
Good start.
There is an issue on the board step. A scrollbar appear when it's displayed.

Capture d’écran du 2022-10-27 21-53-23

**Comment by Ujjawal3** [comment]
@llaske Thanks for reviewing my PR. I have made the required changes and removed the scrollbar.

**Comment by llaske** [comment]
It solve the issue but I'm not a big fan of hard coding x, y shift. The hard coding of toolbar size (55px) is enough.
I suggest instead to forbid scrollbars by using CSS styles:
```

body {
	overflow-x: hidden;
	overflow-y: hidden;
} 
```

**Comment by Ujjawal3** [comment]
@llaske I have done the required changes.

**Comment by llaske** [comment]
Nice.
Thanks @Ujjawal3

**Pull request merged**